.github/await_deployment.py

```diff
@@ -0,0 +1,46 @@
+import time
+import requests
+import os
+
+
+def wait_for_deployment(timeout):
+    branch_name = os.getenv('BRANCH_NAME')
+    print(f"Wait for a deployment for a branch name: {branch_name}")
+    time.sleep(30)
+    bearer_token = os.getenv('VERCEL_TOKEN')
+    project_id = "prj_80UzRn6hG8nvUJSwujWUqqho3osT"
```

Comment:
Should we add it inside the env vars?

Reply:
yes, we can. Better to keep it sealed :)

Reply:
updated!